packages/tools/pie-icons-webc/build.js

```diff
@@ -0,0 +1,114 @@
+import pieIcons from '@justeattakeaway/pie-icons';
+import { pascalCase } from 'pascal-case';
+import path from 'path';
+import fs from 'fs-extra';
+import { execSync } from 'child_process';
+import kebabCase from 'kebab-case';
+import { removeHyphenBeforeDigits } from './helpers/index.js';
+
+const { icons } = pieIcons.default;
+
+const componentTemplate = (name, svg) => `
+import { getDefaultIconSize, iconSize, getSvgProps } from '@justeattakeaway/pie-icons-configs/configs';
+
+const template = document.createElement('template');
+template.innerHTML = '${svg}';
+
+export default class ${name} extends HTMLElement {
+    constructor () {
+        super();
+        const clone = template.content.cloneNode(true);
+
+        this.root = this.attachShadow({ mode: 'open' });
+        this.root.append(clone);
```

Comment:
How come we aren't using Lit?

Reply:
Nothing to add from me – just gets rid of a dependency for simple WCs when we probably don't need it here. If we find it causes us issues down the line though we can always revisit